alfred-telemetry-solr/alfred-telemetry-solr4/build.gradle

```diff
@@ -22,6 +22,5 @@ dependencies {
 createDockerFile {
     from "${solrBaseImage}"
     smartCopy "${rootProject.projectDir}/integration-tests/src/test/resources/${solrFlavor}/solrconfig.xml", "/opt/alfresco/solr4/workspace-SpacesStore/conf/solrconfig.xml"
-    smartCopy "${rootProject.projectDir}/integration-tests/src/test/resources/solr-shared/95-init-solr-micrometer-metrics.sh", "/docker-entrypoint.d/"
```

Comment:
why was this removed?
btw, I don't believe we need Solr4 anymore, both Alfresco 6 and 7 are on ASS, which uses Solr 6 AFAIK